src/main/java/com/altenheim/kalender/controller/viewController/SearchViewController.java

```diff
@@ -59,30 +87,66 @@ else if (button.equals(btnBack) && userStep == 1)
         
         int currentIndex = userStep - 1;
         int requestedIndex = userStep - 1 + incrementor;
-        changeViewState(allSteps[currentIndex], allSteps[requestedIndex]); 
-        changeStepsButtonState(images[currentIndex], images[requestedIndex]);        
+        changeViewState(allSteps[currentIndex], allSteps[requestedIndex], images[currentIndex], images[requestedIndex]); 
         userStep += incrementor; 
         txtHeaderStep.setText(headings[currentIndex]);
     }
 
-    private void changeViewState(VBox deactivate, VBox activate)
+    private void changeViewState(VBox deactivate, VBox activate, Circle currentC, Circle nextC)
     {
         deactivate.setDisable(true);
         deactivate.setVisible(false);
         activate.setDisable(false);
         activate.setVisible(true);
+        currentC.setId("customCircleInactive");
+        nextC.setId("customCircleActive");   
     }
 
-    private void changeStepsButtonState(Circle currentC, Circle nextC)
+    @FXML
+    private void testUpdate(ActionEvent event) throws IOException, ClassNotFoundException
+    {   /*
+        var testEntry = entryFactory.createUserSettingsEntry(timeStart.getValue(), timeEnd.getValue());
```

Comment:
Diese Funktionen  gehören hier nicht rein, habe sie aber einfach zum Ausprobieren mangels passender View durch den Buttonklick ausprobiert. Also Daten soeichern (und hier noch zufällig erzeugen erstmal) und laden und ein dynamisches Update der Liste mit den Vorschlägen. Zum Ausprobieren jeweils ein/auskommentieren, Aufruf in der Smart Search View über den Button "SPEICHERN".